### PR TITLE
feature: k8s dashboard part2 --> add terminal components

### PR DESCRIPTION
**Pull request by laojun**
## What this PR does / why we need it:
feature: add  k8s dashboard terminal components

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |

| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
❎ No

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/modules/cmp/pages/cluster-manage/cluster-terminal.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    subProtocol: 'channel',
+  };
+
+  return (
+    <>
+      <Drawer
+        visible={visible}
+        destroyOnClose
+        onClose={() => setVisible(false)}
+        title={`${i18n.t('cluster')} ${clusterName} ${i18n.t('console')}`}
```
Comment (daskyrk) [inline shell/app/modules/cmp/pages/cluster-manage/cluster-terminal.tsx:43]: Don't concatenate words.

File: shell/app/locales/en.json [hunk 2]
```diff
@@ -616,13 +616,15 @@
...
     "cloud resource overview": "cloud resource overview",
+    "cluster {name} console": "cluster {name} console",
     "cluster configuration": "cluster configuration",
     "cluster registration command": "cluster registration command",
     "configure": "configure",
     "configure automatic renewal": "configure automatic renewal",
     "confirm deletion of cloud account?": "confirm deletion of cloud account?",
     "copy command": "copy command",
     "copy to clipboard": "copy to clipboard",
+    "cpu usage rate": "cpu使用率",
```
Comment (McDaddy) [inline shell/app/locales/en.json:627]: why en.json has chinese

File: shell/app/config-page/components/file-editor/file-editor.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+// or later ("AGPL"), as published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import * as React from 'react';
```
Comment (McDaddy) [inline shell/app/config-page/components/file-editor/file-editor.tsx:14]: no need `* as`

**Pull request merged**